ansible/playbook/tasks/update.yml

```diff
@@ -17,17 +17,28 @@
     - name: Check reread results
       debug: var=reread_result.stdout_lines
 
-    # split download and update to produce a bit more of progress output
```

Comment:
Sorry, we still need that. Otherwise, our API tests for updates timeout on our Jenkins 😆

Reply:
Add sleep in this case, because this section downloads all latests rpms from all repos and then update has conflicts between different versions of PS.

Reply:
`download_only` ignores a list of packages? Ansible docs are silent about that.

Reply:
If you want to download not all but only pmm-* packages it is ok.

Reply:
Yes, exactly. Use a whitelist like now (maybe extract it into variable to avoid duplication), but split download and install steps.